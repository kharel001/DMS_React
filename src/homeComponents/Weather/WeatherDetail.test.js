import React from 'react';
import WeatherDetail from './WeatherDetail';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}) => {
    return shallow( < WeatherDetail
     {...props}  />
    )
}


{/* it("Custom Scrollable render", () => {
        let wrapper = setup(dummy_data);
        // expect(wrapper).toMatchSnapshot();
}) */}
it("Snapshot WeatherDetail", () => {
    let wrapper = setup();
    expect(wrapper).toMatchSnapshot();
   
})

