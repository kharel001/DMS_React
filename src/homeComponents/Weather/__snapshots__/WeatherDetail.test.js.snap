// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot WeatherDetail 1`] = `ShallowWrapper {}`;
